cross/xmlrpc-c/Makefile

```diff
@@ -1,8 +1,8 @@
 PKG_NAME = xmlrpc-c
-PKG_VERS = 1.51.06
+PKG_VERS = 1.59.03
```

Comment:
Likewise, there is a 1.60.05 version available.  I believe the PR had been stalled since a little while which may explain slightly older dependency updates.

Reply:
Done